src/server/analytic_services/prometheus_reports/base_prometheus_report.js

```diff
@@ -6,15 +6,15 @@ const config = require('../../../../config.js');
 
 class BasePrometheusReport {
     constructor() {
-        this._registry = new this.prom_client.Registry();
+        this._register = this.prom_client.register;
```

Comment:
can you please explain about this change?

Reply:
@romayalon Here we are creating a new object but we should point to existing register object that hold all the metrics, `this.prom_client.register` also pointing to `Registry` class.

Reply:
why we should point to an existing register object?

Reply:
existing register object holds all metrics, when we create a new one in our code it does not have any metrics and returns null inside the cluster metrics call.